### PR TITLE
test: remove test and designSystem fixture folders

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- after we removed `design system` folder we moved all `feature` files into the `common` folder structure tree in `cypress`
- simplified generel open step
- refactor for select steps
- amend few `feature` files

### Current behaviour

Currently we have all feature files split for the few different folders 

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [x] Add new Cypress test coverage if required
- [x] Carbon implementation matches Design System/designs

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
All CI should pass :)

### CODE REVIEW

File: cypress/support/step-definitions/simple-color-picker-steps.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-} from "../../locators/simple-color-picker";
+import { simpleColorPickerLegend } from "../../locators/simple-color-picker";
 import {
   experimentalSimpleColorPickerInput,
   simpleColorPicker,
 } from "../../locators/advanced-color-picker/index";
 import { commonDataElementInputPreview } from "../../locators";
 import { keyCode } from "../helper";
 
-Then("Simple Color Picker {int} element was picked up", (index) => {
```
Comment (jamime) [inline cypress/support/step-definitions/simple-color-picker-steps.js:12]: Was this unused?

**Review by jamime — APPROVED** [review]

File: cypress/support/step-definitions/simple-color-picker-steps.js (hunk 1 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/simple-color-picker-steps.js:12]: it came back some how after I've already removed once

**Review by edleeks87 — DISMISSED** [review]

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 2]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+  Scenario: Accordion component primary page opened state
+    Given I open "Accordion" component page "default story"
+    When I expand default accordionRow via click
+    Then "Accordion default page" component has no accessibility violations
+
+  @accessibility
+  Scenario Outline: <component> default story page
+    When I open "Test <component>" component page "default"
+    Then "<component>" component has no accessibility violations
+    Examples:
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:108]: Why is there an Examples table with just one example? Is the intention to expand this to include more examples to warrant having a table?

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 3]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+      | story       |
+      | default     |
+      | validations |
+
+  @accessibility
+  Scenario Outline: Button component <story> page
+    When I open "Button" component page "<story>"
+    Then "Button <story> page" component has no accessibility violations
+    Examples:
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:130]: Why are all of the Button stories not listed here?

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 4]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+      | story        |
+      | sizes        |
+      | with icons   |
+      | icons only   |
+      | icon buttons |
+      | full width   |
+
+  @accessibility
+  Scenario: Button component as a sibling story page
+    When I open "Button Test" component page "as a sibling"
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:166]: Why don't we test all of the Button Test stories?

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 5]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+    When I open "<component> Test" component page "visual"
+    Then "<component>" component has no accessibility violations
+    Examples:
+      | component |
+      | Drawer    |
+      | Grid      |
+
+  @accessibility
+  Scenario: Form component with both errors and warnings summary page
+    When I open "Form" component page "with both errors and warnings summary"
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:180]: Why don't we test all of the Form stories?

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 6]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+    Then "Form" component has no accessibility violations
+
+  @accessibility
+  Scenario: Numeral Date component validations page
+    When I open "Numeral Date Test" component page "validations"
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:185]: Why don't we test all of the Numeral Date Test stories?

File: cypress/features/regression/accessibility/accessibilityCommon.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/accessibility/accessibilityCommon.feature:130]: we are still working on this for accessibility changes

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 7]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+    Then "Numeral Date" component has no accessibility violations
+
+  @accessibility
+  Scenario: Popover Container component default page
+    Given I open "Popover Container" component page "default story"
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:190]: Why don't we test all the Popover Container stories?

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 8]
```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
...
+    When I open popover container
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:191]: Should "popover container" be "Popover Container"?

**Review by sianford — DISMISSED** [review]

**Review by sianford — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Pull request merged**